tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Test_TOV.cpp

```diff
@@ -0,0 +1,96 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+#include <limits>
+#include <pup.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/TOV.hpp"
+#include "PointwiseFunctions/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/EquationsOfState/PolytropicFluid.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+#include "tests/Unit/TestingFramework.hpp"
+
+// Analytic solution in Newtonian limit for polytropic TOV integration has R =
+// pi/alpha, M = 4*pi^2*rho0c/(alpha^3)
+// Here rho0c is the central mass density and alpha = sqrt(2pi/K) where K is the
+// polytropic constant
+
+namespace {
+
+double expected_radius() noexcept {
+  const double polytropic_constant = 4.3419;
+  const double alpha = std::sqrt(2 * M_PI / polytropic_constant);
+
+  return M_PI / alpha;
+}
+
+double expected_mass() noexcept {
+  const double polytropic_constant = 4.3419;
+  const double alpha = std::sqrt(2 * M_PI / polytropic_constant);
+  const double central_mass_density = 1.0e-10;
+
+  return 4 * central_mass_density * pow<2>(M_PI) / (pow<3>(alpha));
+}
+
+template <bool IsRelativistic, size_t dim>
+void test_newtonian_tov(
```

Comment:
const everything that is const and rename things as above, and dim->1 and IsRelativistic->false